src/planscape/plan/models.py

```diff
@@ -71,6 +70,23 @@ class GeneratedProjectAreas(models.Model):
         null=True)
 
 
+class ScenarioSet(models.Model):
+    # TODO: Change "null=True" so that owner is not nullable. Currently owner can be null because
+    # we want alpha users to not be signed in.
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, null=True)  # type: ignore
+
+    project = models.ForeignKey(
+        Project, on_delete=models.CASCADE)  # type: ignore
+
+    # Scenario Parameters:
+
+    # TODO: Limit number of allowed priorities
+    priorities = models.ManyToManyField('conditions.Condition')
```

Comment:
I'm unsure where this is documented, but why is there a need for a Scenario set?  could these priorities be stored in the Project?

Reply:
Maybe we should sync on this. I'm using this table to save the inputs to the forsys run w/o patchmax so that a user can regenerate the run w/ the same inputs if they want to. 

I think how we structure this depends on which options we choose here:

For 1 forsys run w/ patchmax:
Each ScenarioSet has priorities w/o ranking 
Each Scenario has priorities w/ ranking and geometry
Each Scenario has a Project/GeneratedProjectAreas 

For 2 forsys runs:
No ScenarioSet? 
Each Project/GeneratedProjectAreas has priorities w/o ranking 
Each Scenario has priorities w/ ranking

For manually drawn project areas + 1 forsys run w/o patchmax:
Each ScenarioSet has priorities w/o ranking 
Each ScenarioSet has 1 Project/GeneratedProjectAreas
Each Scenario has priorities w/ ranking

Reply:
I think it ultimately depends on what our hierarchy of Scenario/Project is. We haven't talked much about it since we started discussing the various forsys options

Reply:
Updated the PR description with our current idea. Lmk what you think!